src/table/components/head/ColumnAdjuster.tsx

```diff
@@ -19,7 +18,7 @@ const ColumnAdjuster = ({ column, isLastColumn, onColumnResize }: AdjusterProps)
   const columnWidths = useContextSelector(TableContext, (value) => value.columnWidths);
   const setColumnWidths = useContextSelector(TableContext, (value) => value.setColumnWidths);
   const tempWidths = useRef({ adjusterHitArea: {}, columnWidth: 0, initX: 0, initWidth: 0 });
-  const borderHeight = rootElement.getBoundingClientRect().height - PAGINATION_HEIGHT - BORDER_WIDTH;
+  const borderHeight = rootElement?.getElementsByTagName?.('table')[0]?.getBoundingClientRect()?.height;
```

Comment:
according to the types, this could be undefined. Should we add a fallback?

Reply:
I do not see the necessity. If it gets undefined, I implement it wrongly.

Reply:
Yes, it was wrong.